collections/src/main/scala/strawman/collection/immutable/Map.scala

```diff
@@ -10,7 +10,28 @@ import scala.{Any, Boolean, `inline`, Int, None, NoSuchElementException, Nothing
 trait Map[K, +V]
   extends Iterable[(K, V)]
      with collection.Map[K, V]
-     with MapOps[K, V, Map, Map[K, V]]
+     with MapOps[K, V, Map, Map[K, V]] {
+
+  /** The same map with a given default function.
+    *  Note: `get`, `contains`, `iterator`, `keys`, etc are not affected by `withDefault`.
```

Comment:
Instead of giving an incomplete list of methods followed by “etc” I would clearly say what’s the impact: it provides an implementation for the `default` method, which is (internally) used only by the `apply` method.